keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,587 @@
+        config.update(
+            {
+                "include_rescaling": self.include_rescaling,
+                "width_coefficient": self.width_coefficient,
+                "depth_coefficient": self.depth_coefficient,
+                "default_size": self.default_size,
+                "dropout_rate": self.dropout_rate,
+                "drop_connect_rate": self.drop_connect_rate,
+                "depth_divisor": self.depth_divisor,
+                "min_depth": self.min_depth,
+                "activation": self.activation,
+                "input_shape": self.input_shape[1:],
+                "input_tensor": self.input_tensor,
+                "stackwise_kernel_sizes": self.stackwise_kernel_sizes,
+                "stackwise_num_repeats": self.stackwise_num_repeats,
+                "stackwise_input_filters": self.stackwise_input_filters,
+                "stackwise_output_filters": self.stackwise_output_filters,
+                "stackwise_expand_ratios": self.stackwise_expand_ratios,
+                "stackwise_squeeze_and_excite_ratios": self.stackwise_squeeze_and_excite_ratios,  # noqa: E501
+                "stackwise_strides": self.stackwise_strides,
+                "stackwise_conv_types": self.stackwise_conv_types,
+            }
+        )
+        return config
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return copy.deepcopy(backbone_presets)
+
+    @classproperty
+    def presets_with_weights(cls):
+        """Dictionary of preset names and configurations that include
+        weights."""
+        return copy.deepcopy(backbone_presets_with_weights)
+
+
+BASE_DOCSTRING = """Instantiates the {name} architecture.
+
+    Reference:
+    - [EfficientNetV2: Smaller Models and Faster Training](https://arxiv.org/abs/2104.00298)
+      (ICML 2021)
+
+    Args:
+        include_rescaling: bool, whether to rescale the inputs. If set
+            to `True`, inputs will be passed through a `Rescaling(1/255.0)`
+            layer.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e. output of `layers.Input()`)
+            to use as image input for the model.
+    Returns:
+      A `keras.Model` instance.
+"""  # noqa: E501
+
+
+class EfficientNetV2SBackbone(EfficientNetV2Backbone):
+    def __new__(
+        cls,
+        include_rescaling=True,
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        **kwargs,
+    ):
+        # Pack args in kwargs
+        kwargs.update(
+            {
+                "include_rescaling": include_rescaling,
+                "input_shape": input_shape,
+                "input_tensor": input_tensor,
+            }
+        )
+        return EfficientNetV2Backbone.from_preset("efficientnetv2-s", **kwargs)
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return {}
```

Comment:
Remember to add all relevant presets. We have been adding the presets with weights applicable to this architecture (e.g., `"efficientnetv2-s_imagenet"` here)

Reply:
I was under the impression these were intentionally empty due to https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py#L441

I see the others are populated; will populate.

Reply:
Done